Scripts/CI/README_Metadata_StyleCheck/entry.py

```diff
@@ -0,0 +1,564 @@
+import os
+import argparse
+import subprocess as sp
+from os import listdir
+from os.path import isfile, join
+import json
+
+def main():
+    # Tell the program that we're using global variables
+    msg = 'Entry point of the docker to run json style check scripts.'
+    parser = argparse.ArgumentParser(description=msg)
+    parser.add_argument('-s', '--string', help='A JSON array of file paths.')
```

Comment:
> A JSON array of file paths.

file path to that actual file including md or json file extension, or to root directory?

Reply:
This is handled by the GitHub Action. `Action.yml` passes `"-s"` and `$${{ inputs.FILE_PATHS }}` to the Dockerfile, then the Dockerfile runs entry.py with those arguments. The argument is something along the lines of `{"input": [filepath0, filepath1, ...]}` but because this is all run in the GitHub Action it is difficult to get under the hood and debug.